Libs/RaptureCommon/src/test/java/rapture/common/RaptureJsonSerializerTest.java

```diff
@@ -0,0 +1,89 @@
+package rapture.common;
+
+import java.math.BigDecimal;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import rapture.common.impl.jackson.JacksonUtil;
+
+public class RaptureJsonSerializerTest {
+
+    class Bar {
+        @JsonProperty
+        @JsonSerialize(using = RaptureJsonSerializer.class)
+        BigDecimal bdval = new BigDecimal("0.00000001");
+
+        @JsonSerialize(using = RaptureJsonSerializer.class)
+        @JsonProperty
+        Double Dval = 0.00000001;
+
+        @JsonProperty
+        @JsonSerialize(using = RaptureJsonSerializer.class)
+        Float Fval = 0.00000001f;
+
+        @JsonSerialize(using = RaptureJsonSerializer.class)
+        @JsonProperty
+        Double Dlval = 1000000000D;
+
+        @JsonProperty
+        @JsonSerialize(using = RaptureJsonSerializer.class)
+        Float Flval = 1000000000f;
+
+        @JsonProperty
+        @JsonSerialize(using = RaptureJsonSerializer.class)
+        int ilval = 1000000000;
+
+        @JsonProperty
+        @JsonSerialize(using = RaptureJsonSerializer.class)
+        Integer Iilval = 1000000000;
+
+        @JsonSerialize(using = RaptureJsonSerializer.class)
```

Comment:
Only for Doubles and Floats.
I haven't found a solution that works for primitives.
Integer values don't seem to need it. 
BigDecimals are handled separately.